src/ray/gcs/tables.cc

```diff
@@ -508,6 +566,7 @@ Status ActorCheckpointIdTable::AddCheckpointId(const JobID &job_id,
 }
 
 template class Log<ObjectID, ObjectTableData>;
+template class Set<ObjectID, ObjectTableData>;
```

Comment:
`template class Log<ObjectID, ObjectTableData>;` isn't needed any more, right?

Reply:
If I remove this line of code, I'll get the following build error:
```
Undefined symbols for architecture x86_64:
  "ray::gcs::Log<ray::UniqueID, ObjectTableData>::Delete(ray::UniqueID const&, std::__1::vector<ray::UniqueID, std::__1::allocator<ray::UniqueID> > const&)", referenced from:
      ray::gcs::TestDeleteKeysFromSet(ray::UniqueID const&, std::__1::shared_ptr<ray::gcs::AsyncGcsClient>, std::__1::vector<std::__1::shared_ptr<ObjectTableDataT>, std::__1::allocator<std::__1::shared_ptr<ObjectTableDataT> > >&) in client_test.cc.o
  "ray::gcs::Log<ray::UniqueID, ObjectTableData>::Delete(ray::UniqueID const&, ray::UniqueID const&)", referenced from:
      ray::gcs::TestDeleteKeysFromSet(ray::UniqueID const&, std::__1::shared_ptr<ray::gcs::AsyncGcsClient>, std::__1::vector<std::__1::shared_ptr<ObjectTableDataT>, std::__1::allocator<std::__1::shared_ptr<ObjectTableDataT> > >&) in client_test.cc.o
  "ray::gcs::Log<ray::UniqueID, ObjectTableData>::Lookup(ray::UniqueID const&, ray::UniqueID const&, std::__1::function<void (ray::gcs::AsyncGcsClient*, ray::UniqueID const&, std::__1::vector<ObjectTableDataT, std::__1::allocator<ObjectTableDataT> > const&)> const&)", referenced from:
      ray::gcs::TestSet(ray::UniqueID const&, std::__1::shared_ptr<ray::gcs::AsyncGcsClient>) in client_test.cc.o
      ray::gcs::TestDeleteKeysFromSet(ray::UniqueID const&, std::__1::shared_ptr<ray::gcs::AsyncGcsClient>, std::__1::vector<std::__1::shared_ptr<ObjectTableDataT>, std::__1::allocator<std::__1::shared_ptr<ObjectTableDataT> > >&) in client_test.cc.o
ld: symbol(s) not found for architecture x86_64
```

Reply:
Ah yeah, I think this is just because `Set` inherits from `Log`. :(